macros/paths_to_conversion.sql

```diff
@@ -238,15 +252,19 @@ You may obtain a copy of the Snowplow Personal and Academic License Version 1.0
         -- updated with mapping as part of post hook on derived conversions table
         ev.stitched_user_id as customer_id,
       {% else %}
-        case when ev.user_id is not null and ev.user_id != '' then ev.user_id
-            else ev.user_identifier end as customer_id,
+        coalesce(um.user_id, ev.user_identifier) as customer_id,
```

Comment:
If user mapping is disabled, should we still consider the `user_id` in the `conversions_source` table to keep the same behaviour from before for the `customer_id`?

Reply:
Sorry my mistake, for some reason I thought that the user mapping can be disabled but it can't (probably intentionally). So this is fine, ignore me.